test/integration/client-side-encryption/client_side_encryption.prose.test.js

```diff
@@ -853,497 +1072,306 @@ describe('Client Side Encryption Prose Tests', metadata, function () {
-                      ]);
-                    }
-                  });
-                // TODO: Do this in the spec-compliant way using bad auth credentials
-                // .then(
-                //   () => {
-                //     if (withExternalKeyVault) {
-                //       throw new Error(
-                //         'expected insert to fail with authentication error, but it passed'
-                //       );
-                //     }
-                //   },
-                //   err => {
-                //     if (!withExternalKeyVault) {
-                //       throw err;
-                //     }
-                //     expect(err).to.be.an.instanceOf(Error);
-                //   }
-                // );
-              })
-              .then(() => {
-                // #. Use ``client_encryption`` to explicitly encrypt the string ``"test"`` with key ID ``LOCALAAAAAAAAAAAAAAAAA==`` and deterministic algorithm.
-                //    If ``withExternalKeyVault == true``, expect an authentication exception to be thrown. Otherwise, expect the insert to succeed.
-                this.commandStartedEvents.clear();
-                return this.clientEncryption
-                  .encrypt('test', {
-                    keyId: externalKey._id,
-                    algorithm: 'AEAD_AES_256_CBC_HMAC_SHA_512-Deterministic'
-                  })
-                  .then(() => {
-                    if (withExternalKeyVault) {
-                      expect(this.commandStartedEvents.events).to.containSubset([
-                        {
-                          commandName: 'find',
-                          databaseName: keyVaultDbName,
-                          command: { find: keyVaultCollName }
-                        }
-                      ]);
-                    } else {
-                      expect(this.commandStartedEvents.events).to.not.containSubset([
-                        {
-                          commandName: 'find',
-                          databaseName: keyVaultDbName,
-                          command: { find: keyVaultCollName }
-                        }
-                      ]);
-                    }
-                  });
-                // TODO: Do this in the spec-compliant way using bad auth credentials
-                // .then(
-                //   () => {
-                //     if (withExternalKeyVault) {
-                //       throw new Error(
-                //         'expected insert to fail with authentication error, but it passed'
-                //       );
-                //     }
-                //   },
-                //   err => {
-                //     if (!withExternalKeyVault) {
-                //       throw err;
-                //     }
-                //     expect(err).to.be.an.instanceOf(Error);
-                //   }
-                // );
-              })
-          );
+    // Case 4.
+    context('Case 4: KMIP', metadata, function () {
```

Comment:
I think I remember Durran and you (@dariakp) discussing these tests at our virtual office meeting today but can we add a comment here explaning why the fourth KMIP test is omitted?  That would make it more clear for future developers

Reply:
those were different tests, but I can see about filing a follow up for this, too